Blaise-Shyaka/Politico
# [#171591752] Create a landing page for the user

# What does this PR do?

It creates a landing page for users, enabling them to have a basic understanding of what the application is about.

# Description of tasks to be accomplished

- Create a page with a short description of the application
- The page should lead the user to sign in and/or sign up pages
- It should be simple and responsive

# How should this be manually tested?

- Clone this repository
- In the template folder, open index.html file
- Check the presence of sign in and sign up buttons, and a short description of the application
- Resize the browser to verify its responsiveness

# What are the relevant pivotal tracker stories?

#171591752

# Screenshots

![Screenshot (2)](https://user-images.githubusercontent.com/52489421/75830457-9a765280-5db9-11ea-9cb3-4322ca0c5dce.png)
![Screenshot (6)](https://user-images.githubusercontent.com/52489421/75830464-9d714300-5db9-11ea-8728-e31d62073a13.png)
![Screenshot (7)](https://user-images.githubusercontent.com/52489421/75830466-9e09d980-5db9-11ea-8fe5-a0ff31add22a.png)
